_nav_pages/02-full-schedule.md

```diff
@@ -8,7 +8,7 @@ permalink: /schedule/
   {% for session in sessions %}
   <div class="session">
     <hgroup>
-      <h2 id="{{ session.id }}" class="session-title">{{ session.title }}</h2>
+      <h2 id="{{ session.anchor }}" class="session-title">{{ session.title }}</h2>
```

Comment:
@dannielle pssssstttt heads up, don't use `id` anymore if you want the thing we painstakingly typed in, instead use `anchor`. in jekyll 3.x, `id` is automagically generated from the filename and you can't override it.